src/vstest.console/CommandLine/Interfaces/IAssemblyHelper.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors
+{
+    using System.Runtime.Versioning;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+    /// <summary>
+    /// Defines interface for assembly properties.
+    /// </summary>
+    internal interface IAssemblyHelper
```

Comment:
Not sure if this logic belongs here. May be an entrypoint should not know how to read PE files.